### ISSUE TITLE
Cannot find module './src/init'

### ISSUE THREAD

**Issue by jhwohlgemuth** [omahajs/generator-omaha#1]
A recent update to [gulp-sourcemaps](https://github.com/floridoo/gulp-sourcemaps) will cause a freshly scaffolded omaha web app to fail with:

```bash
Loading "imagemin.js" tasks...ERROR
>> Error: Cannot find module './src/init'
```

[This issue](https://github.com/floridoo/gulp-sourcemaps/issues/238) provides the details and can be "fixed" with `npm install gulp-sourcemaps@1.6`

Since `gulp-sourcemaps` is a dependency of a dependency (actually, *several* dependencies...) for apps created with generator-omaha, I do not plan on locking gulp-sourcemaps into version 1.6 (that would be a "slippery slope")

I will keep an eye on gulp-sourcemaps and close this issue when the change ([commit](https://github.com/floridoo/gulp-sourcemaps/commit/93db5f5fd3a05c580b891c41771dc14c1ca585ea) already created) has been implemented in a new release.

**Comment by jhwohlgemuth** [comment]
Actually, it seems that gulp-sourcemap issue with this project is confined to the chain:

`grunt-contrib-imagemin` --> `gulp-rename` --> `gulp-sourcemaps@^1.5.0` --> 🔥 

Maybe I should suggest that the `gulp-rename` project should use `gulp-sourcemaps@1.6.0` instead of `gulp-sourcemaps@^1.5.0`...?

**Comment by jhwohlgemuth** [comment]
Fixed with [v1.8.1](https://github.com/floridoo/gulp-sourcemaps/releases/tag/v1.8.1)
